packages/manager/src/features/CloudPulse/Utils/FilterBuilder.ts

```diff
@@ -107,9 +136,10 @@ export const getResourcesProperties = (
     handleResourcesSelection: handleResourceChange,
     label,
     placeholder,
+    required: true,
```

Comment:
why changes needed in resources selection?

Reply:
we need to make region and resources 'required'?

Reply:
I am not able to understand now...